src/components/cards/SightingsCard.jsx

```diff
@@ -73,51 +57,15 @@ export default function SightingsCard({
       name: 'time',
       label: 'Date',
       options: {
-        customBodyRender: value => formatDate(value, true),
-        getStringValue: value => formatDate(value, true),
+        cellRenderer: cellRendererTypes.specifiedTime,
       },
     },
     {
       reference: 'location',
       name: 'formattedLocation',
       label: 'Location',
       options: {
-        customBodyRender: (_, sightingObj) => {
-          const currentSightingLat = get(
-            sightingObj,
-            'decimalLatitude',
-            '',
-          );
-          const currentSightingLong = get(
-            sightingObj,
-            'decimalLongitude',
-            '',
-          );
-          if (!currentSightingLat && !currentSightingLong)
-            return get(
-              sightingObj,
-              'formattedLocation',
-              intl.formatMessage({ id: 'LOCATION_NAME_MISSING' }),
-            );
-          return (
-            <Button
-              display="link"
-              onClick={() => {
-                setGpsCoordiates({
-                  lat: currentSightingLat,
-                  lng: currentSightingLong,
-                });
-                setModalOpen(true);
-              }}
-            >
-              {get(
-                sightingObj,
-                'formattedLocation',
-                intl.formatMessage({ id: 'LOCATION_NAME_MISSING' }),
-              )}
-            </Button>
-          );
-        },
+        cellRenderer: cellRendererTypes.location,
```

Comment:
This PR adds the ability to specify a "cell renderer" when setting up a table. It also provides the first handful of renderers, which are meant to abstract logic that was being repeated in multiple tables. This diff here is the goal: replace complicated customBodyRender functions that are repeated in multiple places with a simple config string